notebook/notebook/notebook.py

```diff
@@ -241,12 +242,12 @@ def notebook_status_from_pod(pod):
                         cookies=cookies) as session:
                     async with session.get(ready_url) as resp:
                         if resp.status >= 200 and resp.status < 300:
-                            log.info(f'GET on jupyter pod {pod_name} succeeded: {resp}')
+                            log.info(f'notebook_status_from_notebook: GET on jupyter pod {pod_name} succeeded: {resp}')
                             status['state'] = 'Ready'
                         else:
-                            log.info(f'GET on jupyter pod {pod_name} failed: {resp}')
+                            log.error(f'notebook_status_from_notebook: GET on jupyter pod {pod_name} failed: {resp}')
```

Comment:
These are not errors.  We fully expect the probe attempts to fail while the notebook is coming up.  Errors are things that human should ultimately have to investigate.